app/presenters/layouts/footer_presenter.rb

```diff
@@ -69,5 +69,10 @@ def privacy_policy?
     def terms_and_conditions?
       SchoolString::TermsAndConditions.saved?(current_school)
     end
+
+    def add_target_blank?(title)
+      return {} if title == "Home" || title == "Dashboard"
```

Comment:
`"Home"` and `"Dashboard"` are also used in the `nav_link` method. I think you can extract them into constants,like
```ruby
HOME_TITLE = "Home".freeze
DASHBOARD_TITLE = "Dashboard".freeze

def add_target_blank?(title)
  return {} if [HOME_TITLE, DASHBOARD_TITLE].include?(title)
   # ...
end
```
to make future changes easier.

Reply:
also, since the method doesn't return boolean value, you shouldn't add the `?` postfix, which is confusing

Reply:
Hi @st0012, 

Thanks for your helpful suggestion.
I would like to understand the purpose of using the constant with `freeze` method is to achieve the goals below ?
1. Make sure the constant will not be modified.
2. Freeze the object so that system will not produce another memory when the same string is called every time.

Reply:
since it's assigned to a constant, the string will only be allocated once. so the main purpose is to prevent that string being mutated unintentionally by other parts of the program.